src/Mailer/Email.php

```diff
@@ -501,7 +501,6 @@ public static function deliver(
         if (is_array($config) && !isset($config['transport'])) {
             $config['transport'] = 'default';
         }
-        /** @var \Cake\Mailer\Email $instance */
```

Comment:
This also kills IDE compatibility completely here

![issue1](https://user-images.githubusercontent.com/39854/66736812-937e9000-ee6a-11e9-85b4-9158115bceba.png)